app/src/main/res/values/strings.xml

```diff
@@ -2,20 +2,69 @@
     <string name="app_name">ObservationApp</string>
     <string name="username">Username</string>
     <string name="password">Password</string>
-    <string name="login">Login</string>
     <string name="dashboard">Dashboard</string>
     <string name="client_name">Client name</string>
     <string name="project_name">Project name</string>
     <string name="structure_name">Structure name</string>
     <string name="stage_floor">Stage/Floor</string>
     <string name="trade_group">Trade Group</string>
     <string name="activity">Activity</string>
+    <string name="newobservation">New Observation</string>
+    <string name="history">History</string>
+    <string name="goodwork">Good Work</string>
+    <string name="reports">Reports</string>
+    <string name="addphoto">Add Photo</string>
+    <string name="CementStoreisdryandleakproof">Cement Store is dry and leak proof</string>
+    <string name="choosefromgallary">Choose From Gallary</string>
+    <string name="AgeofCementusedatsiteasspecified">Age of Cement used at site as specified</string>
+    <string name="HeightofStacknotexceedinglimits">Height of Stack not exceeding limits</string>
+    <string name="CementBagsarenotdirectlyrestingonground">Cement Bags are not directly resting on ground</string>
+    <string name="FIFOsystemimplementFirstInFirstOut">FIFO system implement(First In First Out)</string>
+    <string name="Adequatespaceavailablearoundbetweenstacksandexteriorwallas">Adequate space available around/between stacks and exterior walla</string>
+    <string name="view">View</string>
+    <string name="MaterialNameCement">Material Name:Cement</string>
```

Comment:
key name should start from small letter and if it is contains 2 separate words then should separate using an underscore
like ---- <string name="client_name">Client name</string>